common/lib/dependabot/pull_request_creator/commit_signer.rb

```diff
@@ -21,6 +20,12 @@ def initialize(author_details:, commit_message:, tree_sha:, parent_sha:,
       end
 
       def signature
+        begin
+          require "gpgme"
+        rescue LoadError
+          raise LoadError, "Please install `gpgme` to enable commit signatures"
```

Comment:
```suggestion
          raise LoadError, "Please add `gpgme` to your Gemfile to enable commit signatures"
```

Reply:
Hah, I originally had that, then mentioned gemspecs too, then simplified to this. Happy to go back to that version though if you think it's clearer.

Reply:
Ha! I *think* it's clearer - worried folks wouldn't realise `gpgme` is a rubygem otherwise. Up to you!

Reply:
Yeah I'm flipping back to it for that reason :-)